cmd/retester/retester.go

```diff
@@ -180,12 +244,52 @@ const (
 	retestBackoffHold = iota
 	retestBackoffPause
 	retestBackoffRetest
-
-	maxRetestsForShaAndBase = 3
-	maxRetestsForSha        = 3 * maxRetestsForShaAndBase
+	maxRetestsForShaAndBaseDefault = 3
+	maxRetestsForShaDefault        = 3 * maxRetestsForShaAndBaseDefault
```

Comment:
`maxRetestsForShaAndBaseDefault` and `maxRetestsForShaDefault` comes from the config now.